airbyte-integrations/connectors/source-tiktok-marketing/Dockerfile

```diff
@@ -32,5 +32,5 @@ COPY source_tiktok_marketing ./source_tiktok_marketing
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=1.0.0
+LABEL io.airbyte.version=1.0.1
```

Comment:
this needs to be a minor upgrade because the schema change is not backwards compatible

Reply:
updated